app/helpers/sign_in_helper.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+module SignInHelper
```

Comment:
You can remove this file altogether since the `ApplicationHelper` already defineds `govuk_button_link_to()`.